### ISSUE TITLE
include recommended Java JDK version info

### ISSUE THREAD

**Issue by duncdrum** [eXist-db/website#16]
We are not always on the latest version, at the moment new users have to locate the GitHub readme to find out which version is recommended. Since exist won't run without Java this should figure more prominently on the webpage.

**Comment by dizzzz** [comment]
it is in the documentation app as well :-/

**Comment by duncdrum** [comment]
Yes, so instead of having to update this multiple times across different articles with every release. I'd rather have this on the webpage, where people are likely to first download exist. We can then include generic links to this location in the documentation app. I'm using [https://bintray.com/existdb/releases/exist#read](https://bintray.com/existdb/releases/exist#read) right now.
